src/stacks-hierarchy/Root_TicketInformationScreen.tsx

```diff
@@ -0,0 +1,77 @@
+import {FullScreenView} from '@atb/components/screen-view';
+import {ThemeText} from '@atb/components/text';
+import {RootStackScreenProps} from '@atb/stacks-hierarchy/navigation-types';
+import {
+  getTextForLanguage,
+  PurchaseOverviewTexts,
+  useTranslation,
+} from '@atb/translations';
+import {ScrollView, View} from 'react-native';
+import {StyleSheet} from '@atb/theme';
+import {GenericSectionItem, Section} from '@atb/components/sections';
+import {SectionHeading} from '@atb/stacks-hierarchy/Root_TabNavigatorStack/TabNav_DashboardStack/Dashboard_RootScreen/components/SectionHeading';
+import {TransportationIconBoxList} from '@atb/components/icon-box';
+
+type Props = RootStackScreenProps<'Root_TicketInformationScreen'>;
+
+export const Root_TicketInformationScreen = (props: Props) => {
+  const {t, language} = useTranslation();
+  const styles = useStyle();
+  const fareProductTypeConfig = props.route.params.fareProductTypeConfig;
+  const preassignedFareProduct = props.route.params.preassignedFareProduct;
+
+  return (
+    <FullScreenView
+      headerProps={{
+        title: t(
+          PurchaseOverviewTexts.ticketInformation.informationDetails.title,
+        ),
+        leftButton: {type: 'close'},
+      }}
+    >
+      <ScrollView style={styles.container}>
+        {preassignedFareProduct.productDescription && (
+          <>
+            <SectionHeading>
```

Comment:
I think `ContentHeading` is the component to use for these headings going forwards. [It was created](https://github.com/AtB-AS/mittatb-app/pull/4111) so that these headings have the same look and feel around the app.